src/pages/Page.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+
+import jsonData from '../data.json';
+import Question from '../components/Question';
+import Button from '../components/Button';
+import Loader from '../components/Loader';
+
+const Page = () => {
+  const [question, setQuestion] = useState([]);
```

Comment:
should be questions 
as it multiple of them not singular

by default it should be null